products/harbor.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Harbor
+category: server-app
+tags: cncf docker registry
+iconSlug: harbor
+permalink: /harbor
+changelogTemplate: https://github.com/goharbor/harbor/releases/tag/v__LATEST__
+releaseDateColumn: true
+eolColumn: End of general support
+
+auto:
+-   git: https://github.com/goharbor/harbor.git
+
+releases:
+-   releaseCycle: "2.9"
+    releaseDate: 2023-09-15
+    eol: 2024-09-30
+    latest: "2.9.1"
+    latestReleaseDate: 2023-09-30
+
+-   releaseCycle: "2.8"
+    releaseDate: 2023-05-22
+    eol: 2024-05-31
+    latest: "2.8.4"
+    latestReleaseDate: 2024-05-31
+
+-   releaseCycle: "2.7"
+    releaseDate: 2023-01-05
+    eol: 2024-01-31
+    latest: "2.7.4"
+    latestReleaseDate: 2023-11-30
+
+-   releaseCycle: "2.6"
+    releaseDate: 2022-09-01
+    eol: 2023-06-30
+    latest: "2.6.4"
+    latestReleaseDate: 2023-03-14
+
```

Comment:
On https://network.pivotal.io/products/harbor-container-registry/releases there are older releases, glad if you can add them too.

Reply:
Actually, after some thinking, I really doubt that this link is related to the Open source version of the Software. Seems like there is "OpenSource Harbor" (Apache license) and VMWare Harbor container registry (VMWare license) based on the open source version.

Reply:
I would update this PR to option 1 and add OpenSource Harbor later as a separate product with a separate PR. WDYT?

Reply:
@BiNZGi I did the title change and added all versions from VMWare page.